### ISSUE TITLE
Implement Marketing Committee UI recommendations

### ISSUE THREAD

**Issue by shabnammayel** [eclipse-ee4j/starter#279]
To improve the look and feel of the [Starter page ](https://start.jakarta.ee), Marketing Committee is recommending the following design to be implemented on this page: 

https://www.figma.com/file/AyF9I5nTEzICVoPORmWp1m/Jakarta-EE-Starter-Page?type=design&mode=design&t=Vv0cCUB75wXlkPWW-1

This is the first time we have provided files in Figma so please let us know if there are access issues.

When you get a chance, can you please review and let me know what you think? thanks

FYI [excentrickristy](https://github.com/excentrickristy) & @TanjaObradovic

**Comment by m-reza-rahman** [comment]
Also perfectly fine to make this change as time and bandwidth allows. Hopefully that will be in a few weeks,

**Comment by kito99** [comment]
@shabnammayel, I'll work on this with @yashTEF after we finish #246.

**Comment by shabnammayel** [comment]
Bumping this up @kito99 would you be able to help us with this and get it implemented within the next month? 

We created a number of Starter guides here last year: https://jakarta.ee/learn/starter-guides/ These guides share screenshots of the starter page and are all referring to the current page design. But we are planning to create more guides this year and potentially some video guides for the Starter. This makes it much more costly and time consuming to go back to all these guides and videos and edit them to refer to the new design once available! That's why I am hoping to get this page updated soon before creating more content pieces! 
If the project team still doesn't have the bandwidth, maybe an option is to get external help to get it implemented. 

Please share your thoughts. thank you

**Comment by m-reza-rahman** [comment]
As I had explained in the marketing committee from the start, there is very limited resources in the project with look and feel skills. In fact in general we have limited resources. Moreover, the project team assessed this as a low priority change as opposed to more critical functionality such as support for Jakarta EE 11. Even from the look and feel standpoint, we have higher priority pending tasks such as including the updated header and footer automatically.

I do not know when we will realistically get to this, but as you can see, it is still in the backlog. Hopefully we will get to it at some point. This is all we can do as a project that is essentially entirely volunteer driven. No one here works on this as part of their day job including me and Kito.

**Comment by kito99** [comment]
@shabnammayel  I finally took a look at this to estimate the required work. I just left a couple of comments on the Figma file; will someone be able to respond there? if not I can leave the questions here instead. 

I'm hoping that @autumnfound will have the updated HTML for the header, footer, and the cards at the bottom, and we just ned to worry about updating the starter UI.

**Comment by m-reza-rahman** [comment]
Hi @kito99, @autumnfound can confirm, but I think the header/footer stuff is sorted now: https://github.com/eclipse-ee4j/starter/issues/246.

**Comment by kito99** [comment]
No worries, @autumnfound. Thanks for the clarification.

**Comment by m-reza-rahman** [comment]
Will take a look ASAP. At the moment, the focus is on EE 11 support in a timely fashion as well as important personal obligations.

**Comment by kito99** [comment]
@shabnammayel @ivargrimstad ok, I made the suggested changes. I added unit tests, but it definitely needs some manual testing.

**Comment by shabnammayel** [comment]
Looks good @kito99 can you please go live with the changes? thanks

**Comment by shabnammayel** [comment]
We want to include the updates to the Starter project as a part of our EE 11 release promotions @kito99 so it would be great to have it live by EOD tomorrow since the official announcement is on 9 a.m. ET 26 June

**Comment by kito99** [comment]
@shabnammayel we need one of the committers to review (I'm not a committer) and approve. cc @m-reza-rahman @ivargrimstad @jeyvison @rokon12

**Comment by kito99** [comment]
@shabnammayel  waiting on Ivar'as review

**Comment by kito99** [comment]
@ivargrimstad this has been reviewed and merged, but it's not live yet. what else needs to happen to deploy it?

**Comment by m-reza-rahman** [comment]
Hi @shabnammayel, @kito99, and @ivargrimstad - I am still struggling to fix all the bugs from the UI refactoring. Part of the issue is that as I have always said, I am no UI expert. I have done what I can but I am now stuck on the issue in the screenshot below. I am not sure, but it looks like a CORS issue with loading CSS from http://jakarta.ee. It's been like this for quite a few months now. For anyone paying attention, it looks careless and unprofessional. Who can help fix this? If I could, I would have already done it. Please help.

<img width="572" height="258" alt="Image" src="https://github.com/user-attachments/assets/224b5a1b-ffe3-4728-9f81-beead26bec3d" />

**Comment by shabnammayel** [comment]
Thanks @m-reza-rahman for your help! I reached out to our webdev team as well and they are looking into it to see if they can help! 
Can you please also check this issue and see if you can help: https://github.com/eclipse-ee4j/starter/issues/368
We are launching the new rebranded website this coming Monday and it would be great to get the starter page updated based on the new look and feel. thanks

**Comment by autumnfound** [comment]
@m-reza-rahman the fix is live, and it looks like it worked like a charm! It was a simple update to our existing font CORS headers to better catch cases like this. I'm guessing this used to work but one of the updates to this site since adding it broke it and we didn't notice at the time.

**Comment by m-reza-rahman** [comment]
Great! Confirmed fixed. Now my question is do we consider this complete or is there something left to do? Who validates and closes this?

**Comment by m-reza-rahman** [comment]
Fixed.
